torch_xla/csrc/tensor.h

```diff
@@ -1426,7 +1430,7 @@ class XLATensor {
       std::string device, ComputationCache::TypePtr cached_computation);
 
   static PostOrderData RunPostOrder(const std::vector<XLATensor>& tensors,
-                                    absl::Span<const size_t> indices);
+                                    SyncTensorCollection* coll);
```

Comment:
@miladm The build error seems maybe due to this header change. I think we need to update all the places where `RunPostOrder` is called in our code.

Reply:
Disregard this, the comment above should fix the build failure I think.